media-tv/xmltv/xmltv-0.6.1.ebuild

```diff
@@ -0,0 +1,301 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit perl-module
+
+DESCRIPTION="Set of utilities to manage TV listings stored in the XMLTV format"
+HOMEPAGE="http://xmltv.org"
+SRC_URI="https://github.com/XMLTV/xmltv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+#KEYWORDS="~amd64 ~arm ~ppc ~x86 ~x86-linux"
+### Temporary until I figure out how to get it fixed!
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+
+IUSE="ar ch-search dk dtvla eu-dotmedia eu-epgdata eu-xmltvse fi fi-sv fr
+huro il is it na-dd na-dtv na-tvmedia pt-meo pt-vodafone se-swedb se-tvzon tr
+tv-check tv-combiner tv-pick-cgi uk-bleb uk-tvguide zz-sdjson zz-sdjson-sqlite"
```

Comment:
Look. ` * Final size of installed tree:   2680 KiB ( 2.6 MiB)` with **every** USE flag enabled. You're making this harder than it needs to be. 

I'd urge you to **at least** combine the channels under one country to a single IUSE. Such as Sweden, Finland, North America, Portugal... there were probably more. (eu + schedulesdirect too)

If it was me, I'd probably just install everything except some tools unconditionally.

Reply:
I tried to combine countries. The configuration is difficult to maintain when grabbers are added/removed by upstream. Different grabbers have different perl module requirements as well.